File: sample-conference-videochat/sample-conference-videochat/Components/UserPorfile/ConferenceUser.m
//
//  ConferenceUser.m
//  sample-conference-videochat
//
//  Created by Injoit on 6/1/20.
//  Copyright © 2020 Quickblox. All rights reserved.
//

#import "ConferenceUser.h"

@implementation ConferenceUser

- (instancetype)initWithID:(NSUInteger)ID fullName:(NSString *)fullName
{
    self = [super init];
    if (self) {
        _ID = @(ID);
        _fullName = fullName;
        _connectionState = QBRTCConnectionStateConnected;
    }
    return self;
}

@end

File: sample-conference-videochat-swift/sample-conference-videochat-swift/Components/Calls/ConferenceUser/ConferenceUser.swift
//
//  ConferenceUser.swift
//  sample-conference-videochat-swift
//
//  Created by Injoit on 07.11.2018.
//  Copyright © 2018 QuickBlox. All rights reserved.
//

import Foundation
import Quickblox
import QuickbloxWebRTC

class ConferenceUser {
    //MARK - Properties
    private var user: QBUUser!
    var connectionState: QBRTCConnectionState = .connecting
    var userName: String {
        return user.fullName ?? CallConstants.unknownUserLabel
    }
    
    var userID: UInt {
        return user.id
    }

    var bitrate: Double = 0.0

    //MARK: - Life Cycle
    required init(user: QBUUser) {
        self.user = user
    }
}
